src/mesh/setup.py

```diff
@@ -26,7 +26,7 @@
 ]
 
 DEPENDENCIES = [
-    'sfmergeutility'
+    'sfmergeutility==0.1.5'
```

Comment:
Add a line in the the readme file about locking the version of this and bump the patch version.